styles.css

```diff
@@ -0,0 +1,963 @@
+	color: #fff;
+	font-size: 0.75rem;
+	font-weight: bold;
+}
+
+.card-discount-badge {
+	position: absolute;
+	right: -1.1rem;
+	top: 0.9rem;
+	text-align: center;
+	width: 5rem;
+	transform: rotate(45deg);
+	background: var(--color-warning);
+	padding: 0.1rem;
+	color: #fff;
+	font-size: 0.7rem;
+	font-weight: bold;
+}
+
+/*Text Cards*/
+/* text card */
+.txt-card {
+	width: 100%;
+	padding: 1rem;
+	margin: 0.5rem;
+	box-shadow: 13px 13px 21px -5px var(--color-grey-bg);
+	border: 1px solid var(--color-grey-bg);
+	/* 6px 6px 6px 3px white; */
+}
+
+.txt-card .title {
+	font-size: 1.5rem;
+	color: var(--text-color);
+	margin: 0;
+	font-weight: 600;
+	margin-bottom: 0.4rem;
+}
+
+.txt-card .subtitle {
+	font-size: 1rem;
+	color: var(--color-grey-dark);
+	margin: 0;
+	margin-bottom: 0.7rem;
+}
+
+.txt-card .txt-content {
+	font-size: 0.7rem;
+	color: var(--color-grey-dark);
+	margin-bottom: 0.4rem;
+}
+
+/*Progress Bar*/
+.progress {
+	height: 1rem;
+	width: 100%;
+	background-color: var(--color-grey-bg);
+	margin: 0.5rem;
+}
+.progress-bar{
+	height: 1rem;
+	margin: 0;
+	background-color: var(--color-primary);
+}
+
+
+/* form */
+.form-container {
+	min-width: 40vw;
+	box-sizing: border-box;
+	padding: 0.5rem;
+	max-width: 600px;
+}
+
+.input-area {
+	resize: none;
+	width: 100%;
+	padding: 0.5rem 1rem;
+	border-radius: 0.2rem;
+	font-size: 1rem;
+	outline: none;
+}
+
+.input-group {
+	position: relative;
+	margin: 0.5rem;
+	width: 100%;
+}
+.input-group .input-area {
+	outline: none;
+	border: 1px solid var(--color-grey);
+}
+
+.input-group label {
+	position: absolute;
+	left: 0.5rem;
+	top: 0.5rem;
+	color: var(--color-grey);
+	background: white;
+	transition: all 0.2s ease-in-out;
+}
+
+.input-group + .err > .input-area{
+	border-color: var(--color-danger);	
+}
+
+.input-group + .err > label{
+	color: var(--color-danger);		
+}
+
+.input-group + .err > .err-msg{
+	color: var(--color-danger);	
+	font-size: 0.7rem;
+}
+
+.input-area:focus {
+	border: 1.5px solid var(--color-primary);
+}
+
+.input-group + .err > .input-area:focus{
+	border: 1.5px solid var(--color-primary);	
+}
+
+.input-area:focus + label {
+	top: -8px;
+	padding: 0 0.45rem;
+	color: var(--color-primary);
+	font-size: 1rem;
+}
+
+/*Basic Forms*/
+
+.basic-form-container {
+	min-width: 40vw;
+	box-sizing: border-box;
+	padding: 0.5rem;
+	max-width: 600px;
+}
+
+.basic-input-group {
+	position: relative;
+	margin: 0.5rem;
+	width: 100%;
+}
+
+.basic-input-group .input-area {
+	outline: none;
+	border: 1px solid var(--color-grey-darkest);
+}
+
+.basic-input-group > label {
+	color: var(--color-grey-darkest);
+}
+.basic-input-group + .err > .input-area{
+	border-color: var(--color-danger);	
+}
+
+.basic-input-group + .err > label{
+	color: var(--color-danger);		
+}
+
+.basic-input-group + .err > .err-msg{
+	color: var(--color-danger);	
+	font-size: 0.7rem;
+}
+
+.basic-input-group + .err > .input-area:focus{
+	border: 1.5px solid var(--color-grey-darkest);	
+}
+
+/* Images
```

Comment:
remove unused(commented) code and minify the css and so it can be loaded faster when in use

Reply:
done👍🏻